packages/web/src/components/premium-content-purchase-modal/PremiumContentPurchaseModal.module.css

```diff
@@ -2,6 +2,10 @@
   width: 720px;
 }
 
+.loading {
+  min-height: 920px;
```

Comment:
Double-check this on mobile web? I don't think this is a safe assumption there.
Also wondering why such a tall fixed height for the loading screen? Is this to avoid it changing size once the content opens? If so, could probably check with design to see if that's a big deal. We could always set the height to animate when it changes so it's not so jarring.

Reply:
oh yeah, i think i removed the usage of this in most recent commit. will delete style too.